src/zjs_aio.c

```diff
@@ -142,8 +142,7 @@ static void ipm_msg_receive_callback(void *context, uint32_t id,
         case TYPE_AIO_PIN_EVENT_VALUE_CHANGE:
             handle->value = (double)pin_value;
             ZVAL num = jerry_create_number(handle->value);
-            zjs_signal_callback(handle->callback_id, &num,
-                                sizeof(jerry_value_t));
+            zjs_signal_callback(handle->callback_id, &num, sizeof(num));
```

Comment:
So this kind of change is just removing some magic; rely on the compiler's knowledge of the type of num, a little cleaner.